.github/workflows/nightly_release.yaml

```diff
@@ -0,0 +1,28 @@
+name: Nightly Release
+
+on:
+  workflow_call:
```

Comment:
let's have this run on a cron job every night at midnight

Reply:
I didn't add the cron job here, since this is a reusable workflow. I added the schedule in the `nightly-release` workflow in k8s-deploy. Is that the right way?